pkg/apis/networking/register.go

```diff
@@ -95,6 +95,7 @@ const (
 	ServiceTypePublic ServiceType = "Public"
 	// ServiceTypeMetrics is the label value for Metrics services. Such services
 	// are used for meric scraping.
+	// TODO(5900): Remove after 0.11 is cut.
```

Comment:
Here and elsewhere 0.12, since we kind of support 10->12 transition.

Reply:
Discussing this separately but doing this for now.